### ISSUE TITLE
Skeletal Animation

### ISSUE THREAD

**Issue by yurembo** [GarageGames/Torque2D#384]
Hello all!
I was played with a skeleton animation.
I created a new small project, included standart "goblins" animation which ships with T2D. Everything is good. This anim runs well. 
But then I exported an animation from new version of Spine. Now Torque 2D is failed.
I think this happens because the export format was changed, am I wrong?

**Comment by yurembo** [comment]
Exactly, old files and new files are different and they count is different

**Comment by capnlove** [comment]
From what I remember, the atlas file added extra information at its beginning. Removing the first line used to do the trick.

Even if it works, remember that the T2D implementation is partial and based on extremely outdated versions of the Spine runtime; most modern Spine features won't be imported to T2D.

**Comment by yurembo** [comment]
Spine runtime is open; I have seen Torque's loader and Spine runtime loader, there are a huge changes

**Comment by yurembo** [comment]
Meanwhile in both old atlas file and new file the first line is empty and second is a name of a graphic file

**Comment by Mpskydog** [comment]
Has there been anyone working on finishing or maintaining any Spine runtime support for this? Nathan Sweet had done some work in it but nothing I can find after this [Trello](https://trello.com/c/koq4dtYk/31-torque2d-runtime) or this [GitHub](https://trello.com/c/koq4dtYk/31-torque2d-runtime)

They've since moved Spine work to a waffle, but no further mention of Torque. And I haven't seen anyone in this community that was actively working on it.

**Comment by greenfire27** [comment]
Nobody has stepped forward to own Spine support. As I understand it is pretty out-of-date at this point.  You are welcome to take up the challenge.

**Comment by yurembo** [comment]
I saw docs on Spine site and looked making Spine support is less simple than T2D has. 
Shoud I dig in :)
